ChangeLog

```diff
@@ -1,5 +1,9 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xxxx-xx-x
+   * Fix compilation warnings with IAR toolchain, on 32 bit platform.
```

Comment:
Could you please this change as a `Bugfix`similar to the previous changes? https://github.com/ARMmbed/mbedtls/blob/development/ChangeLog#L39